src/components/ui/FiltersPart.tsx

```
@@ -27,44 +31,97 @@ const tabletModels = ["iPad Pro", "iPad Air", "Samsung Tab"];
     <span className="text-sm text-gray-700">{label}</span>
   </label>
 );
 
-const SearchIcon = () => <svg className="h-4 w-4 text-gray-500" fill="none" stroke="currentColor" viewBox="0 0 24 24"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" /></svg>;
+const SearchIcon = () => (
+  <svg
+    className="h-4 w-4 text-gray-500"
+    fill="none"
+    stroke="currentColor"
+    viewBox="0 0 24 24"
+  >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)